flask_appbuilder/security/manager.py

```diff
@@ -835,6 +836,7 @@ def _bind_ldap(self, ldap, con, username, password):
             log.debug("LDAP bind OK: {0}".format(username))
             return True
         except ldap.INVALID_CREDENTIALS:
+            log.info("LDAP bind failure: invalid credentials")
```

Comment:
Could be unnecessary at this point since at we have the following:
https://github.com/dpgaspar/Flask-AppBuilder/blob/b4d0db9afa9639f86b201231584e7107810fc526/flask_appbuilder/security/manager.py#L903

and btw, this log at 903 should be a warning, can I ask you to change that also?

Reply:
That log will be emitted in both cases (user not found & password incorrect). I am specifically interested in a log message that is only emitted in the "user found, password incorrect" case.